packages/app/src/components/choropleth-tile.tsx

```diff
@@ -70,13 +70,13 @@ export function ChoroplethTile({
           {typeof description === 'string' ? <Text>{description}</Text> : description}
 
           {onChartRegionChange && chartRegion && (
-            <Box display="flex" justifyContent={{ _: 'center', lg: 'flex-start' }} pt={4}>
+            <Box display="flex" justifyContent={'flex-start'} pt={4}>
```

Comment:
I think we can rewrite it like this.

```suggestion
<Box display="flex" justifyContent="flex-start" paddingTop={space[4]}>
```